packages/template/src/components-page/account-settings/payments/payments-page.tsx

```
@@ -1,20 +1,35 @@
 import { Team, TeamSwitcher } from "../../..";
 import { useUser } from "../../../lib/hooks";
 import { useTranslation } from "../../../lib/translations";
 import { PageLayout } from "../page-layout";
 import { PaymentsPanel } from "./payments-panel";
 
-export function PaymentsPage(props: { mockMode?: boolean }) {
+export function PaymentsPage(props: { mockMode?: boolean, availableTeams?: Team[], allowPersonal?: boolean }) {
   const { t } = useTranslation();
   const user = useUser({ or: props.mockMode ? "return-null" : "redirect" });
-  const teams = user?.useTeams() ?? [];
+  const teams = props.availableTeams ?? user?.useTeams() ?? [];
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)